front/app/components/FormBuilder/components/FormBuilderSettings/ContentSettings/index.tsx

```diff
@@ -93,8 +92,8 @@ export const ContentSettings = ({
               <Toggle
                 name={`customFields.${field.index}.required`}
                 disabled={
-                  !isNilOrError(field.isRequiredEditable)
-                    ? !field.isRequiredEditable
+                  !isNilOrError(lockedAttributes?.required)
```

Comment:
Since `lockedAttributes?.required` cannot be nil or error, we can remove that usage. We could potentially use the lodash get if we want to default or `(lockedAttributes?.required || hasRules)`